python/ray/monitor.py

```diff
@@ -322,6 +325,7 @@ def run(self):
         autoscaling_config = None
 
     monitor = Monitor(
```

Comment:
shouldn't we have to fix test_multinode_2.py since it initializes monitor?

Reply:
This also affects the K8s operator, which also initializes `Monitor`.

Reply:
Ok, I've added the `log_dir=None` by default, which essentially means we just won't benefit from the logging behavior for now (but we still benefit from the node tracking).

Reply:
wait, why log_dir=None and not what it was originally?

Reply:
Realized that /tmp isn't actually a good place to put the logs (it's not actually ideal for anything) and stdout is a reasonable default and the correct behavior for the k8s operator for now